docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -14,76 +14,53 @@
 // Make tab-widgets work
```

Comment:
The `:ot-pipelines:` link on line 7 needs to be updated to `https://opentelemetry.io/docs/collector/configuration/#service`

Reply:
Updated.